skrf/calibration/calibration.py

```diff
@@ -258,7 +258,7 @@ def __init__(self, measured, ideals, sloppy_input=False,
         # ensure that all ideals have same frequency of the measured
         # if not, then attempt to interpolate
         for k in list(range(len(self.ideals))):
-            if self.ideals[k].frequency != self.measured[0]:
```

Comment:
why didnt this  print warnings all over the place before??

Reply:
The frequency class has a `f` property, the network class forwards this property of the underlying frequency object.
So comparing `self.f == other.f` returned true, even if the `other` instance was of type `network`